app/assets/javascripts/admin/addon/components/admin-user-field-item.js

```diff
@@ -64,8 +66,29 @@ export default Component.extend(bufferedProperty("userField"), {
     return ret.join(", ");
   },
 
+  @discourseComputed("buffered.requirement")
+  editableDisabled(requirement) {
+    return requirement === "for_all_users";
+  },
+
   @action
-  save() {
+  changeRequirementType(requirement) {
+    this.buffered.set("requirement", requirement);
+    this.buffered.set("editable", requirement === "for_all_users");
+  },
+
+  async _confirmChanges() {
```

Comment:
I did this wrong in my PR apparently :sweat_smile: I am trying to get clarification from cvx about it. For now this is fine :+1: